CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [4.0.0-alpha1] - 2025-01-30
+
```

Comment:
Please include a few sentences regarding breaking changes and how to resolve them

Reply:
Sorry, I'm not yet done with that and already working on additional info locally.